### ISSUE TITLE
Scanner: Image extraction is not working with GST1.0

### ISSUE THREAD

**Issue by dublok** [mopidy/mopidy#1469]
mopidy-local-images does not extract the cover-art from music files anymore (since GST1.0).

This issue has two sides:

**mopid**
`convert_taglist'() ignores image tags.
https://github.com/mopidy/mopidy/blob/develop/mopidy/audio/tags.py#L52
I miss something like 
```
            elif isinstance(value, Gst.Sample):
                # do the image things here
                result[tag].append(bytes(value.get_buffer())) # ???
```

**mopidy-local-images** 
Does not support GST1.0.
https://github.com/mopidy/mopidy-local-images/blob/develop/mopidy_local_images/library.py#L171
GST1.0 Gst.Buffer has no `data` attribute

**Comment by tkem** [comment]
AFAICS, if image tags are added as `bytes` objects as proposed, mopidy-local-images would fall back on using the object "as is" when there's no `data` attribute.

**Comment by jodal** [comment]
Fixed by PR #1472.
